utils/merge_syft_sbom.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+import json
+import sys
+
+from typing import Any, Callable
+from urllib.parse import urlsplit, quote_plus
+
+
+# Functions to detect components that are reported differently by Syft
+def _is_syft_local_golang_component(component: dict) -> bool:
+    """
+    Check if a Syft Golang reported component is a local replacement.
+
+    Local replacements are reported in a very different way by Cachi2,
+    which is why the same reports by Syft should be removed.
+    """
+    return (
+        component.get("purl", "").startswith("pkg:golang")
+        and component.get("name", "").startswith(".")
+        or component.get("version", "") == "(devel)"
+    )
```

Comment:
Depending on the intended behavior, this may need more parentheses
```python
In [1]: False and False or True
Out[1]: True
```

Reply:
Fixed.